### DISCUSSION TITLE
Best way to delete all files from the database

### DISCUSSION THREAD

**Discussion by jclarke0000** [outdoorbits/little-backup-box#328] [Q&A] [answered]
I've built a backup box and I've opted to go with a large internal SD card for use as internal storage for my backups.  This works well for my purposes as I want to travel with it and I don't want to bring along a number of external devices just to make it work. This setup means I carry just the box, power supply and USB card reader stick.  A very compact setup.

My question is once I return from my travels and I have offloaded all of my photos from the backup box to my permanent storage, what's the best way to clear all of the files off of the internal storage of the backup box? Is there a one-button option to do this?  Could I just go into the file manager and delete everything under "media/internal" or would that cause problems?

**Comment by outdoorbits** [comment, marked as answer]
Sorry for delayed answer. Yes, just delete everything below media/internal using the integrated file browser or whatever you want does the job.

Your setting is compact of course but remember: A power cut in a vulnerable phase of the linux system could damage it and could make it not bootable anymore. Of course, in this case your SD card is still readable by a card reader, your images are not lost. That's why it is recommended to carry at least a second installed SD card when traveling.

**Reply by jclarke0000** [reply]
Thanks!  I'll keep this in mind
